spec/helpers/application_formatters_helper_spec.rb

```diff
@@ -120,6 +120,78 @@ def hello:
           expect(helper.format_html(html)).to be_empty
         end
       end
+
+      context 'when oembed tags are present' do
+        it 'transforms embedded content from youtube' do
+          html = <<-HTML
+            <oembed url="https://www.youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="https://youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="https://m.youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="https://youtu.be/jNQXAC9IVRw"></oembed>
+            <oembed url="http://www.youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="http://youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="http://m.youtube.com/watch?v=jNQXAC9IVRw"></oembed>
+            <oembed url="http://youtu.be/jNQXAC9IVRw"></oembed>
+          HTML
+
+          result = helper.format_html(html)
+
+          expect(result.scan('<iframe').size).to eq(8)
```

Comment:
nit: What do you think about scanning the size for `<oembed` from `html`, similar to how you get the iframe number?